galaxy_ng/app/dynaconf_hooks.py

```diff
@@ -557,6 +557,14 @@ def configure_authentication_backends(settings: Dynaconf) -> Dict[str, Any]:
     if choosen_preset in presets:
         data["AUTHENTICATION_BACKENDS"] = presets[choosen_preset]
 
+    # https://github.com/ansible/django-ansible-base/pull/611
+    data["RENAMED_USERNAME_PREFIX"] = settings.get("RENAMED_USERNAME_PREFIX", "galaxy_")
+    backends = data.get("AUTHENTICATION_BACKENDS", [])
+    backend = "ansible_base.lib.backends.prefixed_user_auth.PrefixedUserAuthBackend"
+    if backend not in backends:
+        backends.insert(0, backend)
```

Comment:
I said this in the eda-server PR too, and from what I understand about this problem, I'd prefer this to be at the _end_ of the list instead of the start. Ideally, this comes into play relatively rarely, and people just login with normal username/password most of the time.